pkg/engine/imageVerify.go

```diff
@@ -148,67 +184,65 @@ type imageVerifier struct {
 
 // verify applies policy rules to each matching image. The policy rule results and annotation patches are
 // added to tme imageVerifier `resp` and `ivm` fields.
-func (iv *imageVerifier) verify(imageVerify kyvernov1.ImageVerification, images map[string]map[string]apiutils.ImageInfo) {
+func (iv *imageVerifier) verify(imageVerify kyvernov1.ImageVerification, imageInfos []apiutils.ImageInfo) {
```

Comment:
maybe rename `imageInfos` to `matchedImageInfos` to clarify that we have already performed the rule matching logic?

Reply:
renamed